### PR TITLE
Windows binary release

### PR DESCRIPTION
**Pull request by stefantalpalaru**
CI: use both cores on GitHub Actions and set timeouts for the local testnet tests

### CODE REVIEW

File: scripts/run-beacon-node.sh [hunk 1]
```diff
@@ -66,7 +75,7 @@ WEB3_HELP
 fi
 
 # Allow the binary to receive signals directly.
-exec build/${NBC_BINARY} \
+exec ${WINPTY} build/${NBC_BINARY} \
```
Comment (zah) [inline scripts/run-beacon-node.sh:78]: Why is this necessary? Another comment above could help.

Comment (stefantalpalaru) [reply scripts/run-beacon-node.sh:78]: Documented here: https://github.com/status-im/nimbus-eth2/pull/2292/files#diff-1c35c3589be12745c19b545aac2e72ff30f7b025ab5d6976734d3f5c5a1317c7R5-R6
